### PR TITLE
refactor(skeleton): migrate arch v18

### PR DESCRIPTION
**Pull request by aesteves60**
First step in skeleton v18 migration

### CODE REVIEW

File: packages/storybook/stories/components/skeleton/1_specifications.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { Meta } from '@storybook/addon-docs';
+import SpecificationsSkeleton from '@ovhcloud/ods-components/src/components/skeleton/documentation/specifications/specifications-skeleton.mdx';
+
+<Meta title="ODS Components/Content/Skeleton/Specifications" />
+
+# Skeleton - Design Specifications
+----
+
+Skeleton TODO add small description:
```
Comment (dpellier) [inline packages/storybook/stories/components/skeleton/1_specifications.stories.mdx:9]: Replace

File: packages/ods/src/components/skeleton/documentation/migration.from.17.x.mdx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+## Attributes changes
```
Comment (dpellier) [inline packages/ods/src/components/skeleton/documentation/migration.from.17.x.mdx:1]: This doc is a really important part of the transition adoption to v18, treat each attribute separately and add a dedicated example for each

File: packages/ods/src/components/skeleton/documentation/migration.from.17.x.mdx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+`inline` <img src="https://img.shields.io/badge/removed-FF0000" />
+`ramdomized` <img src="https://img.shields.io/badge/removed-FF0000" />
```
Comment (dpellier) [inline packages/ods/src/components/skeleton/documentation/migration.from.17.x.mdx:4]: typo: `randomized`

File: packages/storybook/stories/components/skeleton/4_demo.stories.ts [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    <ods-skeleton class="my-skeleton">
+    </ods-skeleton>
+    <style>
+      .my-skeleton::part(skeleton) {
+        ${arg.customCss}
+      }
+    </style>
+  `,
+  argTypes: {
+    customCss: {
```
Comment (dpellier) [inline packages/storybook/stories/components/skeleton/4_demo.stories.ts:18]: good idea 👍, we'll update the previous compo with the same control

**Review by dpellier — CHANGES_REQUESTED** [review]

**Review by astagnol — APPROVED** [review]

**Review by dpellier — CHANGES_REQUESTED** [review]
- style should be more simple
- need only one mixin to obtain skeleton rendering

File: packages/ods/src/style/_skeleton.scss [hunk 5]
```diff
@@ -1,26 +1,13 @@
...
-  }
-}
-
-@mixin ods-skeleton-loader() {
   display: block;
-  position: absolute;
-  top: 50%;
-  right: 0;
-  left: 0;
   transform: translateY(-50%);
```
Comment (dpellier) [inline packages/ods/src/style/_skeleton.scss:3]: should be removed, used previously in conjonction with the `top` value

File: packages/ods/src/style/_skeleton.scss [hunk 6]
```diff
@@ -1,26 +1,13 @@
...
   border-radius: .5rem;
   background-color : var(--ods-color-neutral-050);
   background-image: linear-gradient(-90deg, var(--ods-color-neutral-050), var(--ods-color-neutral-100) 46%, var(--ods-color-neutral-100) 61%, var(--ods-color-neutral-050));
   background-position: -200% 0;
   background-repeat: repeat-y;
   background-size: 50% 12.5rem;
+  width: 100%;
```
Comment (dpellier) [inline packages/ods/src/style/_skeleton.scss:10]: not needed with `display: block` that the default value

**Review by dpellier — CHANGES_REQUESTED** [review]

**Review by dpellier — APPROVED** [review]

**Pull request merged**